quantus/metrics/robustness/continuity.py

```diff
@@ -294,12 +300,12 @@ def evaluate_instance(
 
             # Generate explanation based on perturbed input x.
             dx_step = (step + 1) * self.dx
-            x_perturbed = self.perturb_func(
```

Comment:
Also, why not re-using the class attributes for perturb_func and perturb_func_kwargs, so keeping the self?

Reply:
done